README.md

```diff
@@ -44,6 +44,7 @@ Variable | Default | Description
 `GOOGLE_API_KEY` | N/A | A unique developer [API key](https://developers.google.com/custom-search/json-api/v1/introduction#identify_your_application_to_google_with_api_key) is required to use Google's Custom Search API
 `GOOGLE_CUSTOM_SEARCH` | N/A | The [Google Custom Search][googlesearch] engine [identifier](https://cse.google.com/cse/all) (the `cx` portion of the custom search engine URL)
 `BING_SEARCH_API_KEY` | N/A | The [API key][cognitivebingkey] for performing [Bing Search API][cognitivebing] queries
+`MAX_SEARCH_RESULTS` | 5 | The number of search results to show at maximum
```

Comment:
I think we could simplify the phrasing here. Possibly _"The number of search results to return"_? What are your thoughts? It just seems slightly redundant to say "maximum".